src/components/account/login.vue

```diff
@@ -146,6 +201,14 @@ export default {
       "alreadyLoggedIn": "A user is already logged in. Please refresh the page to continue.",
       "changePassword": "Your password is shorter than 10 characters. To protect your account, please change your password to make it longer."
     },
+    "oidc": {
+      "body": "Click Continue to proceed to the login page.",
+      "error": {
+        "auth-ok-user-not-found": "There is no Central account associated with your email address. Please ask your Central administrator to create an account for you to continue.",
+        "email-not-verified": "Your email address has not been verified by your login server. Please contact your server administrator.",
+        "provider-misconfigured": "Central could not access the email address associated with your account. This could be because your server administrator has configured something incorrectly, or has not set an email address for your account. It could also be the result of privacy options that you can choose during the login process. If so, please try again and ensure that your email is shared."
```

Comment:
This needs to change to `email-claim-not-provided` to account for possibility suggested in https://github.com/getodk/central-backend/pull/910/files#r1289487577